numba/core/extending.py

```diff
@@ -55,7 +55,8 @@ def generic(self):
 _overload_default_jit_options = {'no_cpython_wrapper': True}
 
 
-def overload(func, jit_options={}, strict=True, inline='never'):
+def overload(func, jit_options={}, strict=True, inline='never',
+             prefer_literal=False):
```

Comment:
should this be documented? OR given the likelihood of it becoming a namedtuple or something for the next release, leave it?

Reply:
Actually, i should document it and leave a note that it might change.

Reply:
done in https://github.com/numba/numba/pull/6058/commits/89a3840a6bcc92b30fba70f92653f9eb9c8d66a5